specs/packages/sdk.md

```diff
@@ -44,3 +44,36 @@ Sdk struct wraps logic from the other modules, creating public API using `wasm_b
 
 At the moment, the client of the Sdk is responsible for keeping storage in sync with the Sdk's wallet instance. What does that mean? The Namada extension has its own key storage, which is modified by user actions. Every time the extension storage changes (for example, when a new address is derived), we must manually sync the extension and Sdk wallet storage
 using functions provided by `wallet` module. This is something that will most likely change in the future.
+
+## How can you import and use the SDK?
+First you need to build the shared package. We do that using `wasm-pack`.
+You can take a look at [build.sh](https://github.com/anoma/namada-interface/blob/main/packages/shared/scripts/build.sh) and wasm-pack specific configuration in [Cargo.toml](https://github.com/anoma/namada-interface/blob/main/packages/shared/lib/Cargo.toml)
+After build is done you should see built files in the `src/shared/*`
+```
+LICENSE_MIT
+package.json
+README.md
+shared.d.ts
+shared.js
+shared_bg.js
+shared_bg.wasm
+shared_bg.wasm.d.ts
+```
+You can take a look at `shared.d.ts` to see what is exported.
+
+Next step would be to import SDK itself.
+For now we wrap all of the SDK logic in the [Sdk struct](https://github.com/anoma/namada-interface/blob/main/packages/shared/lib/src/sdk/mod.rs) and [Query struct](https://github.com/anoma/namada-interface/blob/main/packages/shared/lib/src/query.rs). Main reason we do that is to not pass things like `HttpClient`, `Wallet` and `ShieldedContext` every time we want to submit new transaction or query data. This will most likely change in future as we are sometimes getting "multiple mutable references" Rust error.
```

Comment:
> Main reason we do that is to not pass things like `HttpClient`, `Wallet` and `ShieldedContext` every time we want to submit new transaction or query data. This will most likely change in future as we are sometimes getting "multiple mutable references" Rust error.

I would maybe reword this to something like the following (but you may want to check my accuracy ;) ):

(_In regards to Sdk & Query structs_)
```
Creating instances of the `Sdk` and `Query` structs allows us to persist instances of `HttpClient`, `Wallet`, and `ShieldedContext` for the duration of the service-worker lifetime, making them immediately available to any calls invoked by the extension. Currently, the `namada` SDK requires that these references be _mutable_, which presents an issue when attempting to execute SDK functionality in parallel. In the SDK's current implementation, calls should be made sequentially. 
```